tests/utils.py

```diff
@@ -65,6 +65,7 @@ def module_v2_venv(module_path: str) -> Iterator[env.VirtualEnv]:
                 "-X",
                 "module",
                 "install",
+                "-e",
```

Comment:
Before merging, could you double check that this works for pytest-inmanta? I think it might not.

If it does work, could you make the change there as well? Otherwise we may have to rethink this. I think we should try to keep them in sync.